src/drivers/actuators/voxl_esc/Kconfig

```diff
@@ -0,0 +1,5 @@
+menuconfig DRIVERS_ACTUATORS_VOXL_ESC
+	bool "voxl_esc"
+	default n
```

Comment:
If this driver is voxl specific can you add a `depends on` so that non-voxl boards can't select this.

Reply:
No, it isn't voxl specific. The ESC can be used with any flight controller.